index.js

```diff
@@ -29,13 +29,18 @@ const proxy = new Proxy({
 
 proxy.on('incoming', (data, meta, toClient, toServer) => {
   if (meta.name === 'custom_payload') {
-    if (data.channel === 'badlion:mods') return
+    if (data.channel === 'badlion:mods' || data.channel === 'FML|HS' || data.channel === 'FML') return
     else if (data.channel === 'MC|Brand') data.data = Buffer.from('<XeBungee (git:XeBungee-Bootstrap:1.16-R0.5-SNAPSHOT:a2e1df4)') // hack to re-enable freelook on lunar
   }
   toClient.write(meta.name, data)
 })
 
 proxy.on('outgoing', (data, meta, toClient, toServer) => {
+  if (meta.name === 'custom_payload') {
+    if (data.channel === 'MC|Brand') {
+      data.data = Buffer.from('\7vanilla')
```

Comment:
How did you get this string? I just used pakkit and got this: 
![image](https://user-images.githubusercontent.com/43508353/132787885-8b157bc6-6de5-426b-92eb-74cca48e9404.png)
which isn't equal to that

Reply:
I compared what the vanilla 1.8.9 client sends to the server and mimicked that. The \7 is the 7 buffer at the front, and the rest spells out "vanilla".

Reply:
https://wiki.vg/index.php?title=Plugin_channels#:~:text=this%20is%20%22-,vanilla,-%22%20(which